src/lib/dnssd/platform/Dnssd.h

```diff
@@ -203,14 +203,28 @@ CHIP_ERROR ChipDnssdFinalizeServiceUpdate();
  * @param[in] interface   The interface to send queries.
  * @param[in] callback    The callback for found services.
  * @param[in] context     The user context.
+ * @param[out] browseIdentifier an identifier for this browse operation. This
+ *                              can be used to call ChipDnssdStopBrowse.  Only
+ *                              set on success.  This value remains valid until
+ *                              the browse callback is called with an error or
+ *                              is called with finalBrowse set to true.
```

Comment:
Maybe it is obvious, but the `browseIdentifier ` value will also be invalidated by a `ChipDnssdStopBrowse` call if that occurs before an error or final callback.

Reply:
Good point.  Will update the documentation.

Reply:
Actually, the key here is that `ChipDnssdStopBrowse`, if successful, needs to trigger a callback that has an error status or has finalBrowse set to true.  So this part of the documentation is correct, but the documentation for `ChipDnssdStopBrowse` needs to be clearer.